waku/v2/protocol/filter/waku_filter.go

```diff
@@ -92,29 +92,25 @@ func NewWakuFilter(ctx context.Context, host host.Host, isFullNode bool, log *za
 	wf.wg.Add(1)
 	go wf.FilterListener()
 
-	if wf.isFullNode {
-		wf.log.Info("Filter protocol started")
-	} else {
-		wf.log.Info("Filter protocol started (only client mode)")
-	}
-
+	wf.log.Info("filter protocol started")
 	return wf, nil
 }
 
 func (wf *WakuFilter) onRequest(s network.Stream) {
 	defer s.Close()
+	logger := wf.log.With(logging.HostID("peer", s.Conn().RemotePeer()))
 
 	filterRPCRequest := &pb.FilterRPC{}
 
 	reader := protoio.NewDelimitedReader(s, math.MaxInt32)
 
 	err := reader.ReadMsg(filterRPCRequest)
 	if err != nil {
-		wf.log.Error("error reading request", err)
+		logger.Error("reading request", zap.Error(err))
 		return
 	}
 
-	wf.log.Info(fmt.Sprintf("%s: received request from %s", s.Conn().LocalPeer(), s.Conn().RemotePeer()))
```

Comment:
The remote `peer` attribute is attached to the logger above, the local `node` peer ID should be attached to the central logger from the start, so all entries should have it.